src/NUnitEngine/nunit.engine/Agent/AgentProcess.CreateStartInfo.cs

```diff
@@ -0,0 +1,139 @@
+﻿// ***********************************************************************
+// Copyright (c) 2011–2019 Charlie Poole, Rob Prouse
+//
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+//
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+// ***********************************************************************
+
+#if !NETSTANDARD // Dependency on RuntimeFrameworkService
+using NUnit.Engine.Internal;
+using NUnit.Engine.Services;
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+
+namespace NUnit.Engine.Agent
+{
+    internal partial class AgentProcess
+    {
+        private static ProcessStartInfo CreateAgentProcessStartInfo(TestPackage package, RuntimeFrameworkService runtimeFrameworkService)
```

Comment:
These methods are all static. Instead of breaking them out into a partial, should they be moved into a utility class? I like that they've been broken out though 😄